app/models/gobierto_data/dataset.rb

```diff
@@ -10,6 +10,20 @@ class Dataset < ApplicationRecord
     include GobiertoData::Favoriteable
     include GobiertoAttachments::Attachable
     include GobiertoCommon::Collectionable
+    include GobiertoCommon::Searchable
+
+    multisearchable(
+      against: [:name_translations ,:name_en, :name_es],
+      additional_attributes: lambda { |item|
+        {
+          site_id: item.site_id,
+          title_translations: item.truncated_translations(:name), # localice & trunc :name _tranlations
+          resource_path: "/datos/#{item.slug}",
```

Comment:
In Rails **never** use hardcoded paths, that why routes exist. Check any of the the other models using multisearchable and see how they set `resource_path`

Reply:
agree *thanks*